nuxeo-core/nuxeo-core-test/src/test/resources/OSGI-INF/test-coldstorage-contrib.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.coldstorage.test">
+
+  <extension target="org.nuxeo.ecm.core.blob.BlobManager" point="configuration">
+    <blobprovider name="inMemory">
+      <class>org.nuxeo.ecm.core.blob.InMemoryBlobProvider</class>
```

Comment:
You could probably use `DummyBlobProvider` here, maybe a bit cheaper than the in-memory one and we probably don't care about having 2 different implementations.

In fact, if I'm not mistaken, in the real case, we'll have 2 instances of `S3BlobProvider` with a different bucket property.

Reply:
>In fact, if I'm not mistaken, in the real case, we'll have 2 instances of S3BlobProvider with a different bucket property.
yes. the default one who come from `Amazon S3 Online Storage` and a second one who will be configured  by the customer ...

Reply:
>You could probably use DummyBlobProvider here, maybe a bit cheaper than the in-memory one and we probably don't care about having 2 different implementations.

my idea was to separate the things as we will do it in real case

Reply:
OK let's keep it this way.
In fact, it might even correspond to another use case: having the blobs by default on the file system and transferring them to an S3 Glacier bucket for cold storage.

Reply:
`InMemory` is really cheap, I think we have too many dummy classes where the inmemory ones suffice.

Reply:
yes I agree and I think we have two versions of the dummy once: 
`org.nuxeo.ecm.core.DummyBlobProvider` in `nuxeo-core-test`
`org.nuxeo.ecm.core.blob.DummyBlobProvider` -> `nuxeo-core-api`